api/namex/constants/__init__.py

```diff
@@ -104,10 +104,10 @@ class EntityTypes(AbstractEnum):
     # Used for mapping back to legacy oracle codes
     FIRM = 'FIRM'
     # Continued In Types
-    CONTINUE_IN = 'C'             # Continued In BC Ltd.
-    CONTINUE_IN_CBEN = 'CBEN'     # Continued In Benefit Company 
-    CONTINUE_IN_CCC = 'CCC'       # Continued In Community Contribution Company
-    CONTINUE_IN_CUL = 'CUL'       # Continued In Unlimited Liability Company
+    CONTINUE_IN = 'C'             # Continued In BC Limited Company
+    BEN_CONTINUE_IN = 'CBEN'      # Continued In BC Benefit Company 
+    CCC_CONTINUE_IN = 'CCC'       # Continued In BC Community Contribution Company
+    ULC_CONTINUE_IN = 'CUL'       # Continued In BC Unlimited Liability Company
```

Comment:
I renamed these to be consistent with the enum used by the Entity UIs (and Namerequest UI).

Basically, instead of yet another standalone enum, these now match this:
https://github.com/bcgov/bcrs-shared-components/blob/main/src/modules/corp-type-module/corp-type-module.ts